src/inlineKeyboard/inlineKeyboardHandler.js

```diff
@@ -1,19 +1,19 @@
-import { getDayOfWeek } from '../utils/dayOfWeek.js'
 import { createUrlBtn, getButtonDaysOfWeek } from '../buttons/createBotButtons.js'
+import { getTodaySchedule, getTomorrowSchedule } from '../core/schedule.js'
 
 export default async bot => {
 	const getHandlerInlineKeyboard = {
 		getScheduleToday: async ctx => {
-			ctx.replyWithHTML(await getDayOfWeek(), createUrlBtn())
+			ctx.replyWithHTML(await getTodaySchedule(), createUrlBtn())
 		},
 		getScheduleForTomorrow: async ctx => {
-			ctx.replyWithHTML(await getDayOfWeek(1), createUrlBtn())
+			ctx.replyWithHTML(await getTomorrowSchedule(), createUrlBtn())
 		},
 		getScheduleWeek: ctx => {
-			ctx.replyWithHTML('<b>Выберите день:</b>', getButtonDaysOfWeek())
+			ctx.replyWithHTML('<b>Выберите день:</b>', getButtonDaysOfWeek({ raw: false }))
 		},
 		getScheduleNextWeek: ctx => {
-			ctx.replyWithHTML('<b>Выберите день:</b>', getButtonDaysOfWeek(true))
+			ctx.replyWithHTML('<b>Выберите день:</b>', getButtonDaysOfWeek({ raw: true }))
```

Comment:
`{ raw: true }` was just an example. You should find appropriate name for a param instead of `raw`
I meant that every function that used to take a boolean param i.e.

```ts
function myFunction(boolOption) {
  return boolOption ? "foo" : "bar"
}
myFunction(true) // <- this is bad as readability is poor
```

```ts
function myFunction(options) {
  const { preferFoo } = options // or directly destructure in param
  return preferFoo ? "foo" : "bar"
}
myFunction({ preferFoo: true }) // <- better readability
```